src/config/gobjects.rs

```diff
@@ -287,6 +290,17 @@ fn parse_object(
                 .collect()
         })
         .unwrap_or_else(|_| Vec::new());
+    let align = toml_object
+        .lookup("align")
+        .and_then(|v| v.as_integer())
+        .and_then(|v| {
+            if v.count_ones() != 1 || v > i64::from(u32::max_value()) || v < 0 {
+                warn!("align configuration must be a power of two of type u32, found {}", v);
```

Comment:
Please use "\`align\` configuration ..." like in https://github.com/gtk-rs/gir/blob/master/src/config/error.rs#L62